components/applications-service/pkg/server/server.go

```diff
@@ -236,18 +236,17 @@ func convertStorageServicesToApplicationsServices(svcs []*storage.Service) []*ap
 	services := make([]*applications.Service, len(svcs))
 	for i, svc := range svcs {
 		services[i] = &applications.Service{
-			SupervisorId:        svc.SupMemberID,
-			Release:             svc.FullReleaseString(),
-			Group:               svc.Group,
-			HealthCheck:         convertHealthStatusToProto(svc.Health),
-			Status:              applications.ServiceStatus(applications.ServiceStatus_value[svc.Status]),
-			Application:         svc.Application,
-			Environment:         svc.Environment,
-			Fqdn:                svc.Fqdn,
+			SupervisorId: svc.SupMemberID,
+			Release:      svc.FullReleaseString(),
+			Group:        svc.Group,
+			HealthCheck:  convertHealthStatusToProto(svc.Health),
+			Status:       applications.ServiceStatus(applications.ServiceStatus_value[svc.Status]),
+			Application:  svc.Application,
+			Environment:  svc.Environment, Fqdn: svc.Fqdn,
```

Comment:
This some auto-formatting mishap?

Reply:
yup.

Reply:
I can try to bypass it and make app env and fqdn one single line... 🤔

Reply:
Changed!